Makefile

```diff
@@ -20,8 +20,8 @@ OUT ?= bin/dividat-driver
 # Get version from git
 VERSION := $(shell git describe --always HEAD)
 
-# set the channel name to the branch name
-CHANNEL := $(shell git rev-parse --abbrev-ref HEAD)
+# Only channel is main now
+CHANNEL := main
```

Comment:
The information can be updated as well with `make deploy`:

```
	# Print information about channel heads and confirm
	@echo "Channel 'master' is at:"
	@echo "  $(shell git show --oneline --decorate --quiet $$(curl -s "$(RELEASE_URL)master/latest" | tr -d '\n') | tail -1)"
	@echo "Channel 'develop' is at:"
	@echo "  $(shell git show --oneline --decorate --quiet $$(curl -s "$(RELEASE_URL)develop/latest" | tr -d '\n') | tail -1)"
	@echo
	@echo "About to deploy $(VERSION) to '$(CHANNEL)'. Proceed? [y/N]" && read ans && [ $${ans:-N} == y ]
```

Reply:
Thanks, I pushed an updated 47fcc67e99f7fe2c8c40c80bc98029ff3d0994a4.